### PR TITLE
Feature: APP-1255 - SCC Empty State Modal

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- Adds empty state modal for Smart Contract Composer
- Adds demo route `http://localhost:3000/#/scc`

~~NOTE: Branch is based on #550, hence all the duplicated commits. Can hold off on review until base is merged and this one rebased.~~

Task: [APP-1255](https://aragonassociation.atlassian.net/browse/APP-1255)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

**Review by RakeshUP — APPROVED** [review]

File: packages/web-app/src/locales/en/common.json [hunk 1]
```diff
@@ -708,5 +708,13 @@
       "description": "Ready to distribute tokens or pay members? Initiate a token transfer here. For ideas on how to distribute your community's token, read our guide on token distribution.",
       "buttonLabel": "Initiate Transfer"
     }
+  },
+  "scc": {
+    "emptyState": {
+      "modalTitle": "Smart Contract Composer",
```
Comment (sepehr2github) [inline packages/web-app/src/locales/en/common.json:714]: the title is "External Smart Contract" on Figma :)

**Review by sepehr2github — COMMENTED** [review]
Thanks, Fabrice, Looks good. Just a minor thing left

File: packages/web-app/src/locales/en/common.json (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/locales/en/common.json:714]: That's not exactly the case...are we looking at different designs?
image

**Review by sepehr2github — APPROVED** [review]
Thanks Fabrice

File: packages/web-app/src/locales/en/common.json (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/locales/en/common.json:714]: Aha, so we were looking at different designs. Slight difference between desktop and mobile ones 👍🏾

**Pull request merged**